apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -209,6 +230,57 @@
   public static final String ENV_VAR_KEY_RDA_JOB_STARTING_MCS_SEQ_NUM =
       "RDA_JOB_STARTING_MCS_SEQ_NUM";
 
+  /**
+   * The name of the environment variable that should be used to provide the {@link
+   * gov.cms.bfd.pipeline.rda.grpc.RdaServerJob.Config.ServerMode} value for the in-process RDA API
+   * server.
+   */
+  public static final String ENV_VAR_KEY_RDA_GRPC_INPROC_SERVER_MODE =
+      "RDA_GRPC_INPROC_SERVER_MODE";
+
+  /**
+   * The name of the environment variable that should be used to provide the run interval in seconds
+   * for the in-process RDA API server job.
+   */
+  public static final String ENV_VAR_KEY_RDA_INPROC_SERVER_RUN_INTERVAL =
+      "RDA_GRPC_INPROC_SERVER_INTERVAL_SECONDS";
+
+  /**
+   * The name of the environment variable that should be used to provide the random number generator
+   * for the PRNG used by the the in-process RDA API server job's random mode.
+   */
+  public static final String ENV_VAR_KEY_RDA_INPROC_SERVER_RANDOM_SEED =
+      "RDA_GRPC_INPROC_SERVER_RANDOM_SEED";
```

Comment:
Did you want the names of the constants to exactly mirror their value?  Or did you want the environment variable names themselves to be changed?  I was trying to keep the names of the constants shorter since they already have the `ENV_VAR_KEY_` prefix making them unwieldy.

For the environment variable names I used RDA_GRPC for variables specific to the RDA GRPC call configuration (host, port, timeouts, etc) and RDA_JOB for variables specific to the pipeline job but not to the GRPC connection (run interval, enabled, batch size, etc).

Reply:
I don't have any issue with the env var names themselves, just would like that the java variable be consistent with the name since I think that is what folks would expect and might end up wasting time if that assumption does not hold (i.e. assuming they can copy paste and remove the first part to get the variable name).

Reply:
I have corrected the constant names as requested.